File: untitled folder/app/src/main/java/hootor/com/workmanagetdemo/MainActivity.java
package hootor.com.workmanagetdemo;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;

import androidx.work.OneTimeWorkRequest;
import androidx.work.WorkManager;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        final OneTimeWorkRequest oneTimeWorkRequest =
                new OneTimeWorkRequest.Builder(DemoWorker.class).build();


        Button button = findViewById(R.id.button);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                WorkManager.getInstance().enqueue(oneTimeWorkRequest);
            }
        });
    }
}

File: Quiz1C/app/src/main/java/com/android/hootor/quiz1c/MainActivity.kt
package com.android.hootor.quiz1c

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import androidx.navigation.fragment.findNavController
import androidx.navigation.ui.setupWithNavController
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        setSupportActionBar(toolbar)

        bottomNavigationView.setupWithNavController(navHostFragment.findNavController())

        navHostFragment.findNavController()
            .addOnDestinationChangedListener { controller, destination, arguments ->
                when (destination.id) {
                    R.id.settingsFragment, R.id.listConfigurationFragment, R.id.statisticFragment,
                    R.id.syncFragment ->
                        bottomNavigationView.visibility = View.VISIBLE
                    else -> bottomNavigationView.visibility = View.GONE
                }

                when (destination.id) {
                    R.id.listConfigurationFragment -> {
                        tvToolbarTitle.text = "Тесты"
                        tvToolbarTitle.textAlignment = View.TEXT_ALIGNMENT_CENTER
                        appBarLayout.visibility = View.VISIBLE
//                        toolbar.visibility = View.VISIBLE
                    }
                    else -> {
                        appBarLayout.visibility = View.GONE
//                        toolbar.visibility = View.GONE
                    }
                }
            }

    }
}